generators/entity-client/templates/angular/src/test/javascript/e2e/entities/entity.spec.ts.ejs

```diff
@@ -86,61 +86,55 @@ describe('<%= entityClass %> e2e test', () => {
-        expect(<%= entityInstance %>UpdatePage.get<%= fieldNameCapitalized %>Input()).toContain('2001-01-01T02:30');
-        <%_ } else if (fieldType === 'Boolean') { _%>
-        <%= entityInstance %>UpdatePage.get<%= fieldNameCapitalized %>Input().isSelected().then((selected) => {
-            if (selected) {
-                <%= entityInstance %>UpdatePage.get<%= fieldNameCapitalized %>Input().click();
-                expect(<%= entityInstance %>UpdatePage.get<%= fieldNameCapitalized %>Input().isSelected()).toBeFalsy();
-            } else {
-                <%= entityInstance %>UpdatePage.get<%= fieldNameCapitalized %>Input().click();
-                expect(<%= entityInstance %>UpdatePage.get<%= fieldNameCapitalized %>Input().isSelected()).toBeTruthy();
-            }
-        });
-        <%_ } else if (['byte[]', 'ByteBuffer'].includes(fieldType) && fieldTypeBlobContent === 'text') { _%>
-        <%= entityInstance %>UpdatePage.set<%= fieldNameCapitalized %>Input('<%= fieldName %>');
-        expect(<%= entityInstance %>UpdatePage.get<%= fieldNameCapitalized %>Input()).toMatch('<%= fieldName %>');
-        <%_ } else if (['byte[]', 'ByteBuffer'].includes(fieldType)) { _%>
-        <%= entityInstance %>UpdatePage.set<%= fieldNameCapitalized %>Input(absolutePath);
-        <%_ } else if (fieldIsEnum) { _%>
-        <%= entityInstance %>UpdatePage.<%=fieldName %>SelectLastOption();
-        <%_ } else if (fieldType === 'UUID'){ _%>
-        <%= entityInstance %>UpdatePage.set<%= fieldNameCapitalized %>Input('64c99148-3908-465d-8c4a-e510e3ade974');
-        expect(<%= entityInstance %>UpdatePage.get<%= fieldNameCapitalized %>Input()).toMatch('64c99148-3908-465d-8c4a-e510e3ade974');
-        <%_ } else { _%>
-        <%= entityInstance %>UpdatePage.set<%= fieldNameCapitalized %>Input('<%= fieldName %>');
-        expect(<%= entityInstance %>UpdatePage.get<%= fieldNameCapitalized %>Input()).toMatch('<%= fieldName %>');
-        <%_ } _%>
-        <%_ }); _%>
-        <%_ relationships.forEach((relationship) => {
-            const relationshipType = relationship.relationshipType;
-            const ownerSide = relationship.ownerSide;
-            const relationshipName = relationship.relationshipName;
-            const relationshipFieldName = relationship.relationshipFieldName; _%>
-        <%_ if (relationshipType === 'many-to-one' || (relationshipType === 'one-to-one' && ownerSide === true)) { _%>
-        <%= entityInstance %>UpdatePage.<%=relationshipName %>SelectLastOption();
-        <%_ } else if ((relationshipType === 'many-to-many' && ownerSide === true)) { _%>
-        // <%= entityInstance %>UpdatePage.<%=relationshipName %>SelectLastOption();
-        <%_ } _%>
-        <%_ }); _%>
-        <%= entityInstance %>UpdatePage.save();
-        expect(<%= entityInstance %>UpdatePage.getSaveButton().isPresent()).toBeFalsy();
+        <%= entityInstance %>ComponentsPage.clickOnCreateButton().then (
```

Comment:
use async/await where possible

Reply:
As protractor made its own promise wrappers, I choose to use them as I think they're made for something. But I could have been wrong ;-)

Reply:
I have been using async/await with protractor for quite some time and didnt have any issues, so unless there is any recommendation from them to use promise instead of asyc, i would prefer to stick with async/await as they are neater and easier to read